tool_commands/guess_indicators/test_guess_indicators.py

```diff
@@ -0,0 +1,238 @@
+
+
+def get_all_manholes(session):
+    """ get all manholes from sqlite (also manhole_outlet, manhole_pump)
+    :return: a list with tuples (manhole.id, manhole.manhole_indicator) """
+    manholes_id_indicator = {}
+    sql_manholes = session.query(Manhole)
+    for manhole in sql_manholes:
+        # fill dict
+        manholes_id_indicator[manhole.id] = manhole.manhole_indicator
+    return manholes_id_indicator
+
+
+def get_manholes(session, outlet_bool=False, pump_bool=False, all_bool=False):
+    """ get manholes from sqlite: OR only manhole_outlet, OR only manhole_pumps, OR
+    all manholes (manhole_outlet + manhole_pump + other manholes)
+    :return: a list with tuples (manhole.id, manhole.manhole_indicator)  """
+    # only 1 of kwargs may be True
+    assert sum([outlet_bool, pump_bool, all_bool]) == 1
+    if pump_bool:
+        ids = session.query(Pumpstation.connection_node_start_id)
+    elif outlet_bool:
+        ids = session.query(BoundaryCondition1D.connection_node_id)
+    if pump_bool or outlet_bool:
+        manholes_id_indicator = get_pump_or_outlet_manholes(session, ids)
+    if all_bool:
+        manholes_id_indicator = get_all_manholes(session)
+    return manholes_id_indicator
+
+
+def get_all_pipes(session):
+    pipes = {}
+    # only return manhole
+    sql_pipes = session.query(Pipe)
+    for pipe in sql_pipes:
+        # fill dict
+        pipes[pipe.id] = (pipe.friction_type, pipe.friction_value, pipe.material)
+    return pipes
+
+
+# def get_all_pipes_friction_type(session):
+#     pipes_id_friction_type = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_type[pipe.id] = pipe.friction_type
+#     return pipes_id_friction_type
+#
+#
+# def get_all_pipes_friction_value(session):
+#     pipes_id_friction_value = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_value[pipe.id] = pipe.friction_value
+#     return pipes_id_friction_value
+
+
+def empty_manhole_indicator(session, manholes_pre_empty):
+    """ empty column manhole_indicator (set is to NULL) of table v2_manhole """
+    manholes_ids = list(manholes_pre_empty)
+    up = (
+        update(Manhole)
+        .where(Manhole.id.in_(manholes_ids))
+        .values(manhole_indicator=None)
+    )
+    session.execute(up)
+    session.commit()
+
+
+def empty_pipe_friction_value(session, pipes_pre_empty):
+    """ empty column manhole_indicator (set is to NULL) of table v2_manhole """
+    pipes_ids = list(pipes_pre_empty)
+    up = (
+        update(Pipe)
+        .where(Pipe.id.in_(pipes_ids))
+        .values(friction_value=None, friction_type=None)
+    )
+    session.execute(up)
+    session.commit()
+
+
+@pytest.fixture()
+def tmp_sqlite_path(tmpdir):
+    sqlite_filename = "v2_bergermeer.sqlite"
+    orig_sqlite_path = TEST_DATA_DIR.joinpath(
```

Comment:
Die TEST_DATA_DIR was volgens mij al een Path, toch? 
Dan kan dit ook:

    orig_sqlite_path = TEST_DATA_DIR / "testmodel" / "v2_bergermeer" / "sqlite_filename"

De joinpath() werkt ook, dus 't is meer dat je weet dat dit kan. Ik begin langzamerhand aan deze stijl te wennen en het is wel lekker :-)